### PR TITLE
[CS WR] zip up the final xml

### PR DESCRIPTION
**Pull request by bwreid**
[Pivotal Card 94345396](https://www.pivotaltracker.com/story/show/94345396)

This PR allows us to zip the final XML that is outputted by data-processor.

### CODE REVIEW

File: src/vip/data_processor/s3.clj [hunk 1]
```diff
@@ -47,7 +51,13 @@
...
-    (put-object (str "processed-feeds/" new-filename)
-                (.toFile (:xml-output-file ctx)))
-    (assoc ctx :generated-xml-filename new-filename)))
+  (let [zip-filename (str (xml-filename ctx) ".zip")
+        xml-file (-> ctx :xml-output-file .toFile)
+        zip-file (ZipFile. zip-filename)
+        zip-params (doto (ZipParameters.)
+                     (.setCompressionLevel
+                      Zip4jConstants/DEFLATE_LEVEL_NORMAL))]
+    (.createZipFile zip-file xml-file zip-params)
```
Comment (tank157) [inline src/vip/data_processor/s3.clj]: So is this going to write a file, and if so, do we need to do any cleanup? I assume it's in a tmp folder type situation, but even those can fill up.

Comment (bwreid) [inline src/vip/data_processor/s3.clj]: If so, wouldn't we need to do the same for the .xml file? If that's the case, we may want to add a step to clean up all generated files.

Comment (tank157) [inline src/vip/data_processor/s3.clj]: Yeah, I was sort of thinking for both files, I should have made that clear.

**Comment by tank157** [comment]
LGTM with the file cleanup taken care of in another PR.

File: src/vip/data_processor/s3.clj (hunk 1 above)
Comment (bwreid) [inline src/vip/data_processor/s3.clj]: Created [this Pivotal Card](https://www.pivotaltracker.com/story/show/95561156) to address removing these files.

**Pull request merged**